### PR TITLE
Added new post

### PR DESCRIPTION
**Pull request by dbbhattacharya**
Mozilla Kolkata Community Meet Up blog

### CODE REVIEW

File: _posts/2015-09-25-mozilla-kolkata-community-meet-up.markdown [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+The monthly On-Boarding call(Time and Date: 9.00 - 9.30 PM IST / Last Friday of every month) and Online Community Meeting(Time and Date: 9.00 - 9.30 PM IST / Last Tuesday of every month) comes as a real boost to our ambitions of expanding the Community and assigning specific Contribution areas to specific Community members. Not only does it improve Communication among Community members but also provides the very necessary platform to exchange ideas and views alike. We would like all Community members to be a part of these Community meetings and be a part of one of the fastest growing Mozilla Communities in the nation.
+Quick Contact:
+The MozWiki page hosts the list of Community memebers to get in touch with in cases of Contribution queries/event requests/event proposals:
+FSA/Promotion/Hosting Event & Meetup/PR/Marketing/ Event Logistics: Biraj Karmakar
+Contribution Areas Leaders:
+Swaranava Sengupta
+Debapriya Bhattacharya
+Ayan Chowdhury
```
Comment (TechnoAyan) [inline _posts/2015-09-25-mozilla-kolkata-community-meet-up.markdown]: @dbbhattacharya Change my title spelling.. Its 'Choudhury'

**Comment by swarnava** [comment]
@dbbhattacharya lets use our own title instead of Mozilla Kolkata title
from now on.

**Comment by dbbhattacharya** [comment]
@TechnoAyan Priyansu is the writer, not me. but dont worry, i'll make it correct.

@swarnava  i'd love to that but not without others permission

**Comment by TechnoAyan** [comment]
@dbbhattacharya done.. :)

**Comment by dbbhattacharya** [comment]
@TechnoAyan we need edit it again. Seems few spaces are required between the words. :package: 

Have a look : http://mozillakolkata.github.io/blog/jekyll/2015/09/25/mozilla-kolkata-community-meet-up/

**Comment by swarnava** [comment]
please build and test locally before sending the PR, that would help to
avoid error and mistakes!

Thanks!

**Comment by TechnoAyan** [comment]
@dbbhattacharya build it properly.. before sending and preview it..

**Comment by birajkarmakar** [comment]
Thanks @dbbhattacharya  for PR of this long waited blog post. 

Thanks Priyansu for writing this blog.

BTW @swarnava  and all 

>@dbbhattacharya lets use our own title instead of Mozilla Kolkata title
>from now on.

I guess that would be better when to use mozilla Kolkata, when you are not writing any personal blog here. 

As for event reports and monthly blog, would be good to use Mozilla Kolkata. 

But I would recommend to mention those people names who are helping at the bottom of the post;
Like for this case you can write 

"Priyansu, Debopriyo
on the behalf of mozilla kolkata"
Hope this sounds good to you.

**Comment by anistark** [comment]
@birajkarmakar :+1: agreed. Whoever is writing can have their name in the authors place.

@dbbhattacharya if priyansu is writing, let him send PR. That way, he'll learn too an you wouldn't be troubled. You can assist him if needed.  :smile:  Great going guys!

Now, someone go ahead and delete the old posts please.

**Comment by birajkarmakar** [comment]
I told that we would be use Mozilla Kolkata in author place when we post general blog regarding monthly   updates, event updates. Other than that, people can use their names in other place.

No Problem.

**Comment by anistark** [comment]
@birajkarmakar :+1: cool.. Sounds good. :smile:

**Comment by anistark** [comment]
@TechnoAyan Before merging, run it first. Otherwise don't merge. Whoever is merging is responsible for the commit as much as the PR sender.

**Comment by TechnoAyan** [comment]
@anistark Roger that.. I do test before merging.. :)

**Comment by dbbhattacharya** [comment]
@TechnoAyan i did preview and it was fine. Don't know how did this happen? :(
@birajkarmakar i've already mentioned Priyansu's name for writing the report.
@anistark  Ok, i'll tell him to send the PR :)

**Pull request merged**